src/Core/Components/MessageBar/Services/IMessageService.cs

```diff
@@ -16,25 +16,43 @@ public interface IMessageService : IDisposable
     Message ShowMessageBar(Action<MessageOptions> options);
 
     /// <summary />
-    Message ShowMessageBar(string message);
+    Message ShowMessageBar(string title);
 
     /// <summary />
-    Message ShowMessageBar(string message, MessageIntent intent);
+    Message ShowMessageBar(string title, bool allowDismiss);
```

Comment:
I don't think we'd like to make it so easy to remove the close icon.

In FluentUI, it's always preferable for the users to be able to close a message if they wish.

However, adding the option in MessageOptions (only) is still a good idea, I think.

Reply:
Ok.. I will remove the overloads.

Reply:
Removed

Reply:
I think the arguments are still in the `MessageService` class, whereas they are no longer in `IMessageService`.

Reply:
Good catch!  Removed.